app/src/main/java/com/randomappsinc/studentpicker/presentation/PresentationActivity.java

```diff
@@ -134,12 +136,20 @@ private void chooseNames() {
             names.setAlpha(0.0f);
             names.setText(chosenNamesText);
 
-            playSound();
+            if (isMediaVolumeMuted()) {
+                handleNamesAnimations(300);
```

Comment:
Why do we need 300ms delay when it's muted? I feel like the names should just animate in ASAP if there's no volume? Also, how did we get 300?